.circleci/config.yml

```diff
@@ -851,33 +801,46 @@ workflows:
                 - /^rc.*/
                 - /^hotfix.*/
 
-  launch-all-app-builds-workflow:
-    # using do-builds param to select which of the two workflows using the build-on-tag-filters
-    # should run; should be this one that just launches the builds or the one actually doing the builds
-    unless: << pipeline.parameters.do-builds >>
-    jobs:
-      - conditionally-launch-build-and-store-all-job: &build-and-store-filters
-          filters:
-            branches:
-              only:
-                - /^rc.*/
-                - /^hotfix.*/
-
```

Comment:
Replaced by new `build-apps-rc-hotfix` workflow. 
`launch-all-app-builds-workflow` is confusing for few reasons: 
  - It launches builds on rc and hotfix branches only on commits. But the name is not telling that.
  - It doesn't call job to build directly inside the workflow. So builds are still happening while workflow has finished.
  - Builds could have been failed while workflow finished without an error.

See screenshot. Angio has failed to build due to a failed test. Workflow was a pass.

<img width="1029" alt="Screenshot 2023-02-16 at 3 44 05 PM" src="https://user-images.githubusercontent.com/35533885/219482848-06c10144-664a-4e29-87df-691eaf8600f8.png">